README.md

```diff
@@ -67,9 +68,12 @@ This can be set as project property.
 ```
 
 
-##### Scala version configuration
+### Scala version configuration
 
-Plugin supports Scala 2.12.8+, 2.13.0+ and 3.2.0+ versions by automatically loading and configuring matching `scalac-scoverage-plugin` Scalac SCoverage Plugin artifact. For this to work Scala version has to be set. It can be done by defining `scalaVersion` plugin configuration parameter or `scala.version` project property. Without this setting, coverage will not be calculated. 
+Plugin automatically loads and configures matching `scalac-scoverage-plugin` artifact when needed. 
+For this to work Scala version has to be set. It can be done by defining `scalaVersion` plugin configuration parameter or `scala.version` project property. 
```

Comment:
```suggestion
For this to work the Scala version has to be set. It can be done by defining the `scalaVersion` plugin configuration parameter or the `scala.version` project property. 
```